examples/stack/dev/notebook1.scala

```diff
@@ -0,0 +1,2 @@
+# Databricks notebook source
```

Comment:
micronit: rename this to `notebook.scala`

Reply:
👍